### ISSUE TITLE
Latest upgrade throws error on zsh

### ISSUE THREAD

**Issue by shinzui** [clvv/fasd#124]
I started getting errors similar to the one bellow on zsh after upgrading fasd to `1.0.1`.

`/var/folders/qd/bnk2ynz90vbgpjgvqgvs5kww0000gn/T//prezto-fasd-cache.501.zsh:compctl:17: unknown condition code:`

**Comment by belak** [comment]
Looks like the same issue as #123

**Comment by Infinisil** [comment]
@shinzui You upgraded to 5.5.1? I did that too but that didn't fix it for me

**Comment by shinzui** [comment]
@Infinisil You need to blow away the cache then upgrade to 5.5.1 to fix the issue.

**Comment by Infinisil** [comment]
@shinzui Yeah that doesn't work for me, and I'm 100% sure I'm using 5.5.1, and a newer version isn't available. Also https://github.com/sorin-ionescu/prezto/issues/1569#issuecomment-383187300 mentions it's a bug in 5.5 and later, which I think would mean 5.5.1. Also [this mail in the mailing list](http://zsh.org/mla/users/2018/msg00312.html) seems to confirm it still doesn't work with 5.5.1. So I'm pretty sure you got something mixed up and it's not fixed after all for you.

**Comment by shinzui** [comment]
The problem resurfaced so I downgraded zsh for now. It does not look like fasd is maintained so I'm going to switch to something else when I get a chance.

**Comment by giorni** [comment]
@shinzui the code that started the problem is still there. I have submitted a patch, waiting for approval. What are the other options you are thinking about evaluating for a switch?

**Comment by belak** [comment]
The main alternatives I know of are z and autojump, but they're relatively limited.

* [z](https://github.com/rupa/z)
* [autojump](https://github.com/wting/autojump)

**Comment by belak** [comment]
It also should be possible to fix this by moving all the `cat <<EOS` calls into strings stored elsewhere. It'll make the code a bit uglier, but it'll be more compatible.

**Comment by shinzui** [comment]
@shinzui I couldn't find a satisfactory replacement, unfortunately. Forking this package and releasing an updated version with a fix seems to be the best path forward.

**Comment by blanorama** [comment]
(just for completeness: there is also [jump](https://github.com/gsamokovarov/jump))
